src/dune

```diff
@@ -16,36 +16,6 @@
 
 (library
  (name checker)
- (public_name src)
- (modules
-  Checker
-  CheckerEntrypoints
-  CheckerTypes
-  Constants
-  CheckerMain
-  Mem
-  Avl
-  Ctez
-  Kit
-  Lqt
-  FixedPoint
-  Parameters
-  Burrow
-  Cfmm
-  CfmmTypes
-  SliceList
-  LiquidationAuction
-  LiquidationAuctionTypes
-  LiquidationAuctionPrimitiveTypes
-  Ptr
-  Ratio
-  BurrowTypes
-  Ligo
-  LigoOp
-  Fa12Interface
-  Fa2Interface
-  Error
-  Common)
```

Comment:
This is a sneaky change that is actually not necessary for this PR.

However, when I was reading the `dune` docs I saw that these are optional. `checker` indeed compiles without, so I went ahead and deleted them.

@dorranh I see that you changed bits and pieces here recently, can you think of a reason to specify the modules explicitly?

Reply:
Nope, I think it should be fine without them. As far as I know, the only changes I've made here recently were to add the SliceList module.